app/services/vote_service.rb

```diff
@@ -8,6 +8,13 @@ def vote!
     ActiveRecord::Base.transaction do
       delete_previous_votes
       create_vote!
+      debate.reload
+      PusherBroadcaster.push!(votes_count: debate.votes.count,
+                              positive_count: debate.positive_count_with_person,
+                              negative_count: debate.negative_count_with_person,
+                              neutral_count: debate.neutral_count,
+                              positive_percent: debate.positive_percent,
+                              negative_percent: debate.negative_percent)
     end
```

Comment:
I would inject `PusherBroadcaster` to service, this way we could swap notification implementation later (or in tests). I would also use some serializer to serialize debate to reduce coupling